src/intents.js

```diff
@@ -165,6 +186,10 @@ export function createService (cozy, intentId, serviceWindow) {
               type: `intent-${intent._id}:error`,
               error: errorSerializer.serialize(error)
             }),
+            setSize: (doc) => setSize({
```

Comment:
The more I think about this method name the more I feel that it is not explicit enough. Calling `intentService.setSize()` give the idea that we are setting the service own size. I suggest something like `setClientDimensions(dimensions)` or `resizeClient(dimensions)`. What's your opinion about that ?